.github/workflows/ci.yml

```diff
@@ -90,3 +89,21 @@ jobs:
         run: npm ci
       - name: Run Tests
         run: ./node_modules/.bin/ember try:one ${{ matrix.try-scenario }}
+
+  ember__jquery:
+    name: "with @ember/jquery"
+    runs-on: ubuntu-latest
+    needs: 'test'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          cache: npm
+      - name: Install Root Dependencies
+        run: npm ci
+      - name: Install App Dependencies
+        working-directory: tests/apps/test-app--jquery
+        run: npm ci
+      - name: Run App Tests
+        working-directory: tests/apps/test-app--jquery
```

Comment:
a proper approach to this could be to turn the repo into a monorepo, but it seems to be a larger step, so let's consider it later